src/Utils/InitialConditions/init_3d_rising_bubble.jl

```diff
@@ -83,17 +98,30 @@ function init_3d_rising_bubble(::Type{FT}, params, thermovar = :ρθ) where {FT}
     # total energy density
     ρe_tot(local_geometry) = ρ(local_geometry) * e_tot(local_geometry)
 
+    # moisture
+    ρq_tot(local_geometry) = ρ(local_geometry) * q_tot(local_geometry)
+
     # horizontal momentum vector
     uh(local_geometry) = Geometry.Covariant12Vector(0.0, 0.0)
 
     # vertical momentum vector
     w(local_geometry) = Geometry.Covariant3Vector(0.0)
 
-    if thermovar == :ρθ
-        return (ρ = ρ, ρθ = ρθ, uh = uh, w = w)
-    elseif thermovar == :ρe_tot
-        return (ρ = ρ, ρe_tot = ρe_tot, uh = uh, w = w)
+    if thermo_style isa Models.PotentialTemperature
+        ic = (ρ = ρ, ρθ = ρθ, uh = uh, w = w)
+    elseif thermo_style isa Models.TotalEnergy
+        ic = (ρ = ρ, ρe_tot = ρe_tot, uh = uh, w = w)
     else
-        throw(ArgumentError("thermovar $thermovar unknown."))
+        throw(ArgumentError("thermovar $thermodynamics unknown."))
     end
+
+    if moist_style isa Models.Dry
+        return ic
+    elseif moist_style isa Models.EquilibriumMoisture
+        merge(ic, (ρq_tot = ρq_tot,))
+        return ic
+    else
+        throw(ArgumentError("moistmodel $moist unknown."))
+    end
```

Comment:
I think it's generally cleaner to construct these `NamedTuple`'s  once. We basically do this in TurbulenceConvection, [here](https://github.com/CliMA/TurbulenceConvection.jl/blob/main/driver/dycore.jl#L72-L73).

Reply:
whether `ic` has the field `ρq_tot` depends on the `moist_style` here. Is there a convenient way to construct it?